ui/app/helpers/utils/metametrics.util.js

```diff
@@ -157,15 +157,19 @@ function composeUrl (config) {
 
   const urlref = previousPath && composeUrlRefParamAddition(previousPath, confirmTransactionOrigin)
 
-  const dimensions = !pageOpts.hideDimensions ? composeCustomDimensionParamAddition({
-    network,
-    environmentType,
-    activeCurrency,
-    accountType,
-    version,
-    numberOfTokens: (customVariables && customVariables.numberOfTokens) || numberOfTokens,
-    numberOfAccounts: (customVariables && customVariables.numberOfAccounts) || numberOfAccounts,
-  }) : ''
+  const dimensions = pageOpts.hideDimensions
+    ? ''
+    : (
```

Comment:
I'm curious about the parentheses introduced here - they don't seem to be required 🤔